apps/arweave/src/ar_http.erl

```diff
@@ -66,44 +66,46 @@ req(Args) ->
 req(Args, ReestablishedConnection) ->
 	StartTime = erlang:monotonic_time(),
 	#{ peer := Peer, path := Path, method := Method } = Args,
-	Response = case catch gen_server:call(?MODULE, {get_connection, Args}, infinity) of
-		{ok, PID} ->
-			ar_rate_limiter:throttle(Peer, Path),
-			case request(PID, Args) of
-				{error, Error} when Error == {shutdown, normal}; Error == noproc ->
-					case ReestablishedConnection of
-						true ->
-							{error, client_error};
-						false ->
-							req(Args, true)
+	case catch ar_rate_limiter:throttle(Peer, Path) of
+		{'EXIT', _} -> {error, client_error};
```

Comment:
When will this clause get triggered? It looks like `throttle` only returns `ok` or ... exits the whole application? (is that right?)